### ISSUE TITLE
Please add the commands in English

### ISSUE THREAD

**Issue by luizfernandonb** [starflash-studios/QMediaVSIX#1]
**Is your feature request related to a problem? Please describe.**
I don't know Japanese

**Describe the solution you'd like**
The extension translated into English, and it is possible to change languages ​​for those who still want Japanese

**Describe alternatives you've considered**
None

**Additional context**
![image](https://user-images.githubusercontent.com/42288418/118895174-0a4bfc80-b8dc-11eb-8f83-b50499a13114.png)

**Comment by Neonalig** [comment]
Ah, yeah. I just realised that I implemented the translation strings wrong, and the Japanese language strings were overriding the English ones.

I just spent the better part of two days trying to work out how to properly add localisation support to the extension, but all methods I happened upon either didn't work, broke the project, or contradicted each other, so I ended up adding a manual translation system.

Since the extension is an AsyncPackage, my scripts are unable (to my knowledge at least) to be ran before the user actually clicks on a MenuItem, so in the new version until I find a relevant fix, chances are if you start the project, the language may be incorrect. If so, just click on one of the buttons (i.e. the info button) and it should update all the titles.
